src/test/java/io/netty/incubator/codec/http3/Http3UnidirectionalStreamInboundHandlerTest.java

```diff
@@ -174,7 +174,7 @@ private void testPushStream(long maxPushId) throws Exception {
             ReferenceCountUtil.release(written);
         }
 
-        if (maxPushId >= 0) {
+        if (!server && maxPushId >= 0) {
```

Comment:
I don't think this is needed once you address my comments on the `Http3ControlStreamOutboundHandler`

Reply:
this is removed